recipes/verta/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "verta" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 24db7f5922fd63db8094717a020560435e121f5bf9c7c9478bad5938fe08947e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python >= 3.5, < 3.8
```

Comment:
Implemented change; removed `< 3.8` and removed spaces after operators.

Preventing installation with 3.8 was mostly just to avoid compatibility bugs when it's released, but I guess most packages aren't worrying about that.